src/components/AccessibilityRequestsTable/index.tsx

```diff
@@ -159,6 +154,8 @@ const AccessibilityRequestsTable: FunctionComponent<AccessibilityRequestsTablePr
     return 'none';
   };
 
+  const tableColWidths = [350, 150, 260, 150, 260]; // [request name, submission date, business owner, test date, status]
```

Comment:
It still feels a bit strange to hard-code these col. widths like this. lmk if there's a better way.

Reply:
Is there a reason why we need to set every single column width? That feels a little overprescribed, but I might not have all the context.

Reply:
IMO, if we do need to set every width, I think doing it inline is okay. Reading these values from an object is an option too. I would stay away from the array.

Reply:
Sai wanted each column width to be relevant to the expected length of the data (for example, the date data length is shorter, while the name will need more width.  Maybe. there's a more dynamic way to do it?  like adding max width for the date columns?

Since we are looping through the columns for each row, I needed to do it dynamically (but maybe there's another way?).  I could try switching to an object, but not sure how I'd identify the key in this case- tips welcome.